src/modules/m_check.cpp

```diff
@@ -295,16 +295,17 @@ class CommandCheck : public Command
 			const user_hash& users = ServerInstance->Users->GetUsers();
 			for (user_hash::const_iterator a = users.begin(); a != users.end(); ++a)
 			{
-				if (InspIRCd::Match(a->second->GetRealHost(), parameters[0], ascii_case_insensitive_map) || InspIRCd::Match(a->second->GetDisplayedHost(), parameters[0], ascii_case_insensitive_map))
+				User* user = a->second;
```

Comment:
`fatal error: declaration shadows a local variable [-Wshadow]`

Reply:
@SadieCat, fix it.
https://github.com/inspircd/inspircd/commit/87f5e7e18f618ebd33c64ec100b2e9be09d1bf0d